src/main/webapp/add_event_page/script.js

```diff
@@ -15,8 +18,35 @@ function initMap(){
         var latLng = marker.latLng; 
         currentLatitude = latLng.lat();
         currentLongitude = latLng.lng();
-        jQ("#latitude").val(currentLatitude);
-        jQ("#longitude").val(currentLongitude);
     }); 
 }
-window.initMap = initMap;
\ No newline at end of file
+window.initMap = initMap;
+
+function submitEvent(){
+    var event_name = document.getElementById("name").value;
+    var event_des = document.getElementById("description").value;
+    var event_loc = document.getElementById("location_name").value;
+    var event_date = document.getElementById("date").value;
```

Comment:
Could change this to "var event_date = new Date(document.getElementById("date").value).toISOString();"?